dsp_permissions_scripts/models/scope.py

```diff
@@ -121,12 +120,12 @@ def remove(
 
 
 PUBLIC = PermissionScope.create(
-    CR={PROJECT_ADMIN},
+    CR={PROJECT_ADMIN, CREATOR},
     D={PROJECT_MEMBER},
-    V={UNKNOWN_USER, KNOWN_USER},
+    V={UNKNOWN_USER},
```

Comment:
If I understand the permissions correctly then known-user does not have any permissions anymore because they are not mentioned right?

Reply:
That is theoretically correct. But according to https://docs.dasch.swiss/latest/DSP-API/02-dsp-ontologies/knora-base/#permissions:

> If the result is that she would have no permissions, give her whatever permission UnknownUser would have.

So in fact, this change is a pure simplification, without actually changing anything.